docs/en/solutions/How_to_Use_Pipeline_to_Train_AI_Models.md

```diff
@@ -0,0 +1,952 @@
+---
+products:
+   - Alauda AI
+kind:
+   - Solution
+ProductsVersion:
+   - 4.x
+---
+# How to Use Pipeline to Train AI Models
+
+## Overview
+
+This document demonstrates how to train AI models using DevOps Pipeline. The YOLOv5 model is used as an example to illustrate the training workflow. The overall framework presented here can be adapted for training other models as well, requiring only adjustments to input parameters and execution scripts.
+
+
+## Prerequisites
+
+Before proceeding with the AI model training pipeline, ensure the following prerequisites are met:
+
+1. **Alauda DevOps**: Install `Alauda DevOps next-gen` following the [devops documents](https://docs.alauda.io/devops). `Alauda DevOps Pipelines` and `Alauda DevOps Connectors` must be installed.
+
+2. **Volcano**: Install the `Volcano` cluster plugin to enable GPU scheduling and resource management for AI workloads.
+
+3. **Storage**: Prepare CSI (Container Storage Interface) storage. A temporary storage space is required during training to store models, datasets, and training data.
+
+4. **Required Repositories**: Prepare:
```

Comment:
- optional: Alauda AI. Install Alauda AI for better management of models, datasets, and create inference services using trained models.
- optional: GPU device plugins